homeassistant/components/fibaro/__init__.py

```diff
@@ -77,18 +78,28 @@
     })
 }, extra=vol.ALLOW_EXTRA)
 
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Optional(CONF_GATEWAYS, default={}):
```

Comment:
default should be a list too?

Reply:
Now this will be converted to be `[{}]`, if that is expected, make that the default.